example/cra-provider/src/Web3Buttons.tsx

```diff
@@ -0,0 +1,84 @@
+import { useWeb3React } from '@web3-react/core'
+import { Connector } from '@web3-react/types'
+import { useEffect, useState } from 'react'
+import { useArcxAnalytics } from '@arcxmoney/analytics'
+import { metamask, walletConnect } from './connectors'
+
+export const Web3Buttons = () => {
+  const sdk = useArcxAnalytics()
+  const { account, connector, chainId, provider } = useWeb3React()
+  const [connectedWallet, setConnectedWallet] = useState<'metamask' | 'walletConnect' | undefined>()
+
+  useEffect(() => {
+    if (sdk && provider) {
+      sdk.setProvider(provider.provider as any)
+    } else if (sdk && sdk.provider && !provider) {
+      // provider disconnected from external source (e.g. user clicked disconnect in wallet connect)
+      setConnectedWallet(undefined)
+    }
+  }, [sdk, provider])
+
+  const onConnectWalletClicked = async (givenConnector: Connector, chainId?: number) => {
+    try {
+      console.log({ givenConnector: givenConnector, provider: givenConnector.provider })
```

Comment:
Did you leave this comment with purpose?

Reply:
no, removed